ao_experimental/quant_primitives.py

```diff
@@ -178,6 +178,222 @@ def dequantize_per_channel(
     return y
 
 
+class DynamicallyQuantizedLinear(torch.nn.Module):
```

Comment:
nit: I'd recommend a separate file for modules, this is a user facing API which is separate from primitives

Reply:
ok, will do